sql/handler.cc

```diff
@@ -6745,6 +6753,14 @@ int ha_abort_transaction(THD *bf_thd, THD *victim_thd, my_bool signal)
   DBUG_RETURN(0);
 }
 
+#ifdef WITH_WSREP
+void wsrep_wait_until_innodb_initialized()
+{
+  handlerton *hton= installed_htons[DB_TYPE_INNODB];
+  hton->wait_until_initialized(hton);
+}
+#endif /* WITH_WSREP */
```

Comment:
Do we really need to add this function?

Reply:
What is suggested alternative?

Reply:
This function does not seem to be always initialized when we arrive here:
Thread 1 "mysqld" received signal SIGSEGV, Segmentation fault.
0x0000555555f84167 in wsrep_wait_until_innodb_initialized () at /home/jan/mysql/10.3e/sql/handler.cc:6854
(gdb) p hton->wait_until_initialized
Cannot access memory at address 0x140
(gdb)

Reply:
./mtr wsrep.mdev_6832 --verbose --mysqld=--wsrep-sync-wait=0 --mysqld=--wsrep-debug=ON --ddd